_posts/2023-07-31-workshops.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Online Training Workshop Series
+---
+
+MDAnalysis is excited to be kicking off a series of free online training workshops in Fall 2023. The first will be held on **October 25, 2023 from 15:00-18:30 UTC**. The next workshop will be organized for Australasian time zones and will be announced shortly.
```

Comment:
Are we 100% on the austalasian edition or should we be saying that we are looking into it?

Reply:
Softened the language since we don't actually have a date on the calendar for this yet